src/frontend/src/handler/create_function.rs

```diff
@@ -187,43 +187,43 @@ pub async fn handle_create_function(
             );
         }
         "rust" => {
-            identifier = wasm_identifier(
-                &function_name,
-                &arg_types,
-                &return_type,
-                matches!(kind, Kind::Table(_)),
-            );
             if params.using.is_some() {
                 return Err(ErrorCode::InvalidParameterValue(
                     "USING is not supported for rust function".to_string(),
                 )
                 .into());
             }
-            let function_body = params
+            let script = params
                 .as_
                 .ok_or_else(|| ErrorCode::InvalidParameterValue("AS must be specified".into()))?
                 .into_string();
-            let script = format!("#[arrow_udf::function(\"{identifier}\")]\n{function_body}");
-            body = Some(function_body.clone());
+            let script = format!("use arrow_udf::{{function, types::*}};\n{}", script);
```

Comment:
> Due to the introduction of the new struct syntax, users now need to write #[function(...)] explicitly (previously we would prepend this macro for them).

What about checks whether user uses `#[function` in the script, and decide whether to added this for them? 🤡 In this way, we can keep the simple old syntax.

Reply:
Sounds a little hacky. But I like this idea. 🤡

Reply:
Final rule:
- if the return type is a struct, they have to write `#[function]` by themselves.
- otherwise, we add it for them.